bindgen/CMakeLists.txt

```diff
@@ -39,8 +39,12 @@ add_executable(bindgen
   Utils.h
   ir/IR.h
   ir/IR.cpp
+  ir/StructOrUnion.cpp
+  ir/StructOrUnion.h
```

Comment:
Following clang's naming convention the file and class could alternatively be called "Record"

Reply:
👍